flytekit/clis/sdk_in_container/package.py

```diff
@@ -0,0 +1,231 @@
+import os
+import sys
+import tarfile
+import tempfile
+import typing
+from collections import OrderedDict
+
+import click
+from flyteidl.admin import launch_plan_pb2, task_pb2, workflow_pb2
+
+from flytekit import LaunchPlan
+from flytekit.clis.sdk_in_container import constants, serialize
+from flytekit.common import translator
+from flytekit.configuration import internal
+from flytekit.core import context_manager
+from flytekit.core.base_task import PythonTask
+from flytekit.core.context_manager import ImageConfig, look_up_image_info
+from flytekit.core.workflow import WorkflowBase
+from flytekit.tools import fast_registration, module_loader
+
+_DEFAULT_IMAGE_NAME = "default"
+_DEFAULT_RUNTIME_PYTHON_INTERPRETER = "/opt/venv/bin/python3"
+
+
+def validate_image(ctx: typing.Any, param: str, values: tuple) -> ImageConfig:
```

Comment:
It feels like this function exist elsewhere, right?

Reply:
Soren added it! https://github.com/flyteorg/flytekit/pull/394/files#diff-ded0a6913453e62ae1a80623d886c46931197ef4d206f0e10155475a8f356df0R68

Reply:
https://github.com/flyteorg/flytekit/pull/394/files#diff-ded0a6913453e62ae1a80623d886c46931197ef4d206f0e10155475a8f356df0R68

Reply:
sadly no. It does use the actual parse image function that exists